src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs

```diff
@@ -80,14 +76,18 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
         }
 
         var text = await document.GetValueTextAsync(cancellationToken).ConfigureAwait(false);
-        var options = (CSharpCodeGenerationOptions)await document.GetCodeGenerationOptionsAsync(context.Options, cancellationToken).ConfigureAwait(false);
```

Comment:
@tmat this is showing up as actually expensive in practice.  the reason being that it seems like no caching is being done, so this always calls through aall the way to:

![image](https://github.com/dotnet/roslyn/assets/4564579/560514f7-0012-4d9f-ad50-3270ec78559a)

Given that these values are not changing during these experiences (scrolling, for example) doing this work every time we call into this refactoring provider is wasteful, and shows up as where roslyn is actually spending its time.